test/e2e/bootstrapper_test.go

```diff
@@ -76,11 +76,11 @@ func TestBootstrapper(t *testing.T) {
 	// Run it again, to ensure it maintains state if the bootstrapper is already started
 	time.Sleep(5 * time.Second)
 	wmcb, err = bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath)
-	assert.Nilf(t, err, "Could not create WinNodeBootstrapper: %s", err)
+	assert.NoErrorf(t, err, "Could not create WinNodeBootstrapper: %s", err)
```

Comment:
This needs to be `require`. I missed this during the initial review but caught it when rebasing #96.